README.md

```diff
@@ -123,4 +123,4 @@ This proposal was created to track the "static" (i.e., of the constructor) aspec
 
 Draft specification text is published at [https://tc39.github.io/proposal-static-class-features/].
 
-Static public fields are implemented in Babel and V8 behind a flag, and had test262 tests written (though currently have been removed when this proposal was demoted to Stage 2). Static private fields are currently [underway in Babel](https://github.com/babel/babel/issues/8052).
+Static public fields are implemented in Babel and V8 behind a flag and had test262 tests written (though currently have been removed when this proposal was demoted to Stage 2). Static private fields are currently [underway in Babel](https://github.com/babel/babel/issues/8052).
```

Comment:
The comma seems correct to me; why remove it?

Reply:
The comma is redundant.

Reply:
How so? It indicates a pause in the thought, and is appropriate to optionally appear before the conjunction.